src/zenml/integrations/mlflow/steps/mlflow_deployer.py

```diff
@@ -82,79 +85,91 @@ def mlflow_deployer_step(
     def mlflow_model_deployer(
         deploy_decision: bool,
```

Comment:
I'll relay this recommendation from Hamza here: you should change the MLflow standard deployer step to also take in a `ModelArtifact`, even if it doesn't use it. Currently, we rely on the `deploy_decision` argument to guarantee that the steps in the pipeline are executed in the correct order: the model needs to be trained and auto-logged in the MLflow store before the deployer step is executed, otherwise we don't have a model to serve here. But using the `deploy_decision` to order steps in a pipeline is awkward. It makes more sense that this step take in a model as an input artifact.

In a follow-up revamp of this deployer, we can perhaps use MLflow to deploy models that are not logged to MLflow, instead of having to base everything on the condition that models are logged to MLflow first.

Reply:
This makes sense. I have added the model to the step's parameters. Thanks!